passkeeper/__init__.py

```diff
@@ -77,6 +77,20 @@ def cleanup_ini(self):
                 LOG.info('Clean file %s' % fname)
                 run_cmd('shred --remove %s' % file_path)
 
+    def cleanup_history(self, directory):
+        "Shred all .git files"
+        for root, dirs, files in os.walk(directory, topdown=False):
```

Comment:
All this part could be in more generic function like `shred_dir`. We could reuse this part of code I think